src/daily-auths-report.tsx

```diff
@@ -122,7 +122,11 @@ function tabulate(
   };
 }
 
-function tabulateSumByAgency(results?: ProcessedResult[], filterIal?: number): TableData {
+function tabulateSumByAgency(
+  results?: ProcessedResult[],
+  filterIal?: number,
+  setParameters?: (params: Record<string, string>) => void
```

Comment:
No need for this to be optional, is there? Saves us a optional chaining operator below as well.

```suggestion
  setParameters: (params: Record<string, string>) => void
```

Reply:
we can't have required args after optional args so it would change the order of the function a bit

Reply:
Ah, I guess I assumed this was an object argument.

Reply:
object argument makes more sense... updated in 317631463d30761b24ca9d10b2235f7ac91d5662